src/sentry/static/sentry/app/actionCreators/tags.jsx

```diff
@@ -4,14 +4,12 @@ import TagStore from 'app/stores/tagStore';
 import TagActions from 'app/actions/tagActions';
 import AlertActions from 'app/actions/alertActions';
 
-const api = new Client();
-
 const MAX_TAGS = 500;
 
 export function fetchTags(orgId, projectId) {
   TagStore.reset();
   TagActions.loadTags();
-
+  const api = new Client();
```

Comment:
I didn't want to break existing usage of this, but these action creators should be given an instance of `Client` so that components can abort xhr on unmount